addons/hr/static/src/views/kanban_view.js

```diff
@@ -5,15 +5,11 @@ import { registry } from '@web/core/registry';
 import { kanbanView } from '@web/views/kanban/kanban_view';
 import { KanbanModel } from '@web/views/kanban/kanban_model';
 
-export class EmployeeKanbanRecord extends KanbanModel.Record {
-    async openChat(employeeId) {
-        this.model.env.services["mail.messaging"].openChat({ employeeId });
-    }
-}
+export class EmployeeKanbanRecord extends KanbanModel.Record {}
 
 export class EmployeeKanbanModel extends KanbanModel {}
 
-EmployeeKanbanModel.services = [...KanbanModel.services, "mail.messaging"];
+EmployeeKanbanModel.services = [...KanbanModel.services];
```

Comment:
Should remove custom kanban + `js_class`

Reply:
done!